godot-macros/src/class/data_models/field_export.rs

```diff
@@ -370,12 +386,32 @@ impl FieldExport {
                 or_greater,
                 or_less,
                 exp,
+                radians_as_degrees,
                 radians,
                 degrees,
                 hide_slider,
-            } => quote_export_func! {
-                export_range(#min, #max, #step, #or_greater, #or_less, #exp, #radians, #degrees, #hide_slider)
-            },
+                suffix,
+            } => {
+                let suffix = if suffix.is_some() {
+                    quote! { Some(#suffix.to_string()) }
+                } else {
+                    quote! { None }
+                };
+                let export_func = quote_export_func! {
+                    export_range(#min, #max, #step, #or_greater, #or_less, #exp, #radians_as_degrees || #radians, #degrees, #hide_slider, #suffix)
+                }?;
+                let deprecation_warning = if *radians {
+                    quote! {
+                    #export_func;
+                    ::godot::__deprecated::emit_deprecated_warning!(export_range_radians);
+                            }
```

Comment:
Indentation

Reply:
This is what rustfmt produces.  It wouldn't pass CI if it weren't like this.

Reply:
Are you sure? Rustfmt is typically rather conservative when it comes to formatting the inside of macro invocations (as in, often doesn't format at all). That's also the value proposition of tools like [Gene Michaels](https://github.com/andrewbaxter/genemichaels).

Reply:
```
$ rustfmt --check godot-macros/src/class/data_models/field_export.rs
Diff in /home/fp/3rd/gdext/godot-macros/src/class/data_models/field_export.rs at line 402:
                 }?;
                 let deprecation_warning = if *radians {
                     quote! {
-			#export_func;
-			::godot::__deprecated::emit_deprecated_warning!(export_range_radians);
-                    }
+                    #export_func;
+                    ::godot::__deprecated::emit_deprecated_warning!(export_range_radians);
+                            }
                 } else {
                     quote! { #export_func }
                 };
```

Reply:
Weirdly, this doesn't produce any diff for me, despite being seemingly the same.
```rs
impl Something {
    fn produce_ugly_diff() {
        match something {
            AnonymouStructVariant {
                f0,
                f1,
                f2,
                f3,
                f4,
                f5,
                f6,
                f7,
                f8,
                f9,
                f10,
            } => {
                let warning = if *radians {
                    quote! {
                        #export_func;
                        ::godot::__deprecated::emit_deprecated_warning!(export_range_radians);
                    }
                } else {
                    quote! { #export_func }
                };
            }
        }
    }
}
```

Reply:
Assuming it's a bug, I've reported it https://github.com/rust-lang/rustfmt/issues/6233

Reply:
Thanks a lot for reporting! 👍 

So for now you can leave the bad formatting, maybe add a comment that links to this conversation:

https://github.com/godot-rust/gdext/pull/783#discussion_r1669105958

Reply:
Done